app/views/assigned_applications/index.html.erb

```diff
@@ -23,6 +23,30 @@
   </div>
 </div>
 
-<% if @applications.any? %>
-  <%= render 'application_table', applications: @applications %>
+<% unless @search.results.empty? %>
+  <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <table class="govuk-table app-dashboard-table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <%= table_headers(
```

Comment:
I think we either use the partial or the helper - either one is fine but I'm not sure we need the partial if we are using it in some places and not others 🤔